webui/react/src/pages/TrialDetails/TrialDetailsLogs.tsx

```diff
@@ -152,6 +152,7 @@ const TrialDetailsLogs: React.FC<Props> = ({ experiment, trial }: Props) => {
         decode(optional(DateString), options.timestampAfter),
         options.orderBy as OrderBy,
         settings.searchText,
+        false,
```

Comment:
just checking: regex is disabled in webui right?

Reply:
Right, it will be enabled in a separate PR